object/service.proto

```diff
@@ -688,3 +692,15 @@ message GetRangeHashResponse {
   // transmission.
   neo.fs.v2.session.ResponseVerificationHeader verify_header = 3;
 }
+
+// TODO: docs
+message ReplicationRequest {
+  // TODO: reserve 1st byte for forward compatibility
+  bytes object = 1;
+
+  neo.fs.v2.refs.Signature object_signature = 2;
```

Comment:
That's the node signature, right? Because there is already one in `Object`.

Reply:
yeah right, we can improve naming, e.g. `object_field_signature`